### PR TITLE
fixes from RR demo

### PR DESCRIPTION
**Pull request by shark8me**
- fix dependency failures due to multiple versions of web3j transitive imports
- add API to create account using a token instead of username/password
- fix path for invoke API in DDO.

### CODE REVIEW

File: src/main/clojure/starfish/core.cljc [hunk 1]
```diff
@@ -261,12 +261,12 @@
 ;; Account
 
 (defn remote-account
-  [^String username ^String password]
-  ;; TODO: shouldn't this 
-  (let [^String id (or username (error "Username required!"))
-        ^java.util.Map creds {"username" username
-                         "password" password}]
-    (RemoteAccount/create id creds)))
+  ([token](RemoteAccount/create (Utils/createRandomHexString 32)
```
Comment (mikera) [inline src/main/clojure/starfish/core.cljc]: I think we should allow passing an arbitrary credential map here? Perhaps as well as a token option

File: src/main/clojure/starfish/core.cljc [hunk 2]
```diff
@@ -318,11 +318,11 @@
   (^Asset [^Operation operation params]
    (let [job (invoke operation params)
          resp (.getResult job)]
-     resp))
+     (into {} resp)))
```
Comment (mikera) [inline src/main/clojure/starfish/core.cljc]: Probably want a keywordize-keys anywhere that we return a map from Java into Clojure land?

**Review by mikera — CHANGES_REQUESTED** [review]

**Comment by mikera** [comment]
Couple of minor comments. Please add tests too! Otherwise good to merge.

File: src/main/clojure/starfish/core.cljc (hunk 1 above)
Comment (shark8me) [reply src/main/clojure/starfish/core.cljc]: @mikera did you mean having accepting both
- token (as a String)
- credential map (as a map).

Something like this doesn't compile due to `Can't have 2 overloads with same arity`
```clj
(defn remote-account
  ([^String token] (RemoteAccount/create (Utils/createRandomHexString 32)
                                {"token" token}))
  ([^clojure.lang.APersistentMap credentials] (RemoteAccount/create (Utils/createRandomHexString 32)
                                credentials))
  ([^String username ^String password]
   (RemoteAccount/create (Utils/createRandomHexString 32)
                         {"username" username
                          "password" password})))
```

**Pull request merged**